src/index.test.js

```diff
@@ -0,0 +1,66 @@
+// import React from 'react';
+import ReactDOM from 'react-dom';
+import Adapter from 'enzyme-adapter-react-16';
+import * as createjs from 'createjs-module';
+import { configure } from 'enzyme';
+import AccessibilityModule from './index';
+
+configure({ adapter: new Adapter() });
+
+describe('AccessibilityModule', () => {
+  describe('setupStage', () => {
+    let canvasEl;
+    let stage;
+    let container;
+    let parentElement;
+    let renderSpy;
+
+    beforeEach(() => {
+      canvasEl = document.createElement('canvas');
+      stage = new createjs.Stage(canvasEl);
+      container = new createjs.Container();
+      parentElement = document.createElement('div');
+      parentElement.id = 'parent-element';
+      renderSpy = jest.spyOn(ReactDOM, 'render');
```

Comment:
Are we expecting to use renderSpy in multiple tests? If not, should it just be moved into the first test rather than beforeEach?

Reply:
it most assuredly will (or should), but yes, we could move it into the one test that uses it right now (a second test had to be nicked)

Reply:
ah gotcha, it seems fine where it is now then!